cluster/src/lib.rs

```diff
@@ -44,6 +47,46 @@ pub type ClusterRef = Arc<dyn Cluster + Send + Sync>;
 
 pub type TableManipulatorRef = Arc<dyn TableManipulator + Send + Sync>;
 
+pub type ClusterTopologyRef = Arc<ClusterTopology>;
+
+pub type TableLocs = HashMap<String, ShardView>;
```

Comment:
what does `TableLocs` stand for

Reply:
Actually, It stands for `TableLocations`, but let me figure out a better name. How about `TableShards`?

Reply:
The `ShardView` has been modified to:
```rust
pub struct NodeShard {
    pub shard: ShardInfo,
    pub node: Node,
}

pub struct TableNodeShards {
    pub table_id: TableId,
    pub node_shards: Vec<NodeShard>,
}
```


And the `TableLocs` should be `HashMap<TableName, TableNodeShards>`. Finally, this type alias is decided to be removed because I guess no need and no good name to alias such a type `HashMap<TableName, TableNodeShards>`.

Reply:
> Finally, this type alias is decided to be removed
👍